### DISCUSSION TITLE
Cutting with the laser

### DISCUSSION THREAD

**Discussion by ghost** [arkypita/LaserGRBL#1194] [Q&A]
Hi! I am totally frustrated!! I have tried so many different settings (suggestions from Ortur and CNC3-3018Pro) , trying to cut through 1,6mm plywood. For example: S1000 with 200mm/min and I should only need one or two passes. After 4 passes and at different sppeds I just burn the wood on the top and nothing goes through. What am I doing wrong?

I can engrave. The picture turned out fine, so I think I focused the laser correctly.

Please help me!!

Thank you!!

SassyLady 59

**Comment by arkypita** [comment]
What kind of laser do you have? Power?

**Comment by ghost** [comment]
Thank you rfor the speedy reply...
I have a 15 w Ortur
For the wood cutting I used 1000 power and 200 mm/ min

**Comment by ghost** [comment]
I just tried to cut through again, to no avail. You can see little scratches on the back side, but not enough so that the pieces of wood pop out.
![IMG_20201208_220149](https://user-images.githubusercontent.com/66275739/101542373-f605ea00-39a2-11eb-8421-512b1c0df559.jpg)
![IMG_20201208_220158](https://user-images.githubusercontent.com/66275739/101542376-f69e8080-39a2-11eb-9085-0c393d3df1b8.jpg)

![IMG_20201208_220216](https://user-images.githubusercontent.com/66275739/101541914-50527b00-39a2-11eb-8277-7000dc3b3fcf.jpg)
![IMG_20201208_220228](https://user-images.githubusercontent.com/66275739/101541938-56e0f280-39a2-11eb-8669-30b26cb16efa.jpg)

**Comment by arkypita** [comment]
First, do not use vectorize for this kind of image. Use centerline instead.

Vectorize is ok with filled figure like this one
![image](https://user-images.githubusercontent.com/8782035/101543660-d1127680-39a4-11eb-99cf-ea046bea4400.png)

But will produce internal and external border (double line) if used on line-make images.

**Comment by arkypita** [comment]
Second: i have ortur too (15 and 20W) there isn't much difference between the two and I honestly prefer the 15W.

With S1000 and F400 i cut poplar plywood up to 5mm in 8-10 passes BUT it is a very light plywood. Other essences and other densities require longer times, the glues used in the production of plywood also greatly influence.

Focus is the key to everything. It must be extremely perfect.
This look focused, other pictures does not look focused.

![image](https://user-images.githubusercontent.com/8782035/101544074-6150bb80-39a5-11eb-8806-e8ad0fc0a175.png)

To achieve optimal focus you need to turn on the laser at the lowest possible power and focus on a non-reflective black material. the ideal is to glue a small piece of black electrician's tape on the plywood to be cut, and focus on it.

The 15W model has an optimal focus distance between 70-80mm

![image](https://user-images.githubusercontent.com/8782035/101544876-b214e400-39a6-11eb-9f2b-9cb9c50efa21.png)

**Comment by arkypita** [comment]
This means that it focus better if you have 7-8cm between your plywood and the laser module

**Comment by ghost** [comment]
Thank you so much. I will try it out on Friday & will let you know what happens!
